theme/overrides/home.html

```diff
@@ -1,5 +1,13 @@
 {% extends "base.html" %}
 
+{% block announce %}
+<div class="md-announce">
```

Comment:
I don't think we need this `<div>` at all, since all `md-announce` class is already managed by the `block announce` above.

Currently this renders an additional black background for the link on top of not-so-black background offered by `md-announce`, which looks weird.

Reply:
Hm. For some reason in the insiders version it's not setting the `md-announce` wrapper. Let me investigate...

Reply:
Workarounded in 9f609ea9bf87f247e2fb1b8ff1c7fcbac72c4b78